### PR TITLE
fix: visibility of custom form field and info text color

### LINKED ISSUE #4607: Attendee Form Design Issue: "custom form" option shows up even if Feature is turned "off" + Info text unreadable
1) The attendee form in wizard step 2 can be switched on or off. 
Expected: The custom form option should only be visible if the attendee form is switched on.

 2) The text below "information about attendee form" is not readable as the grey color is to bright on many screens.

Screenshot from 2020-07-21 16-28-59

### PR DESCRIPTION
**Pull request by ruhci28**
Fixes #4607

#### Short description of what this resolves:
custom form field is visible even when the attendee form is turn off . Text under Information about attendee form is very light. 

#### Changes proposed in this pull request:

-Make the Custom form field visible only when the attendee form is turn on . 
-change the color of the text under Information about attendee form to make it more readable and visible. 
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ x
Screenshot 2020-07-27 at 10 06 48 PM
Screenshot 2020-07-27 at 10 02 53 PM

 I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/forms/wizard/attendee-step.hbs [hunk 1]
```diff
@@ -92,11 +92,13 @@
...
-    @customForms={{this.data.customForms}}
-    @event={{this.data.event}}
-    @form="attendee" />
+  {{#if this.data.event.isTicketFormEnabled}}
+   <Forms::Wizard::CustomFormInput
+     @newFormField={{this.data.newFormField}}
+     @customForms={{this.data.customForms}}
+     @event={{this.data.event}}
+     @form="attendee" />
+  {{/if}}
```
Comment (iamareebjamal) [inline app/templates/components/forms/wizard/attendee-step.hbs:101]: Why add one more if and nor just move it inside the if above?

**Comment by iamareebjamal** [comment]
Also, please change the PR title to something proper

File: app/templates/components/forms/wizard/attendee-step.hbs (hunk 1 above)
Comment (ruhci28) [reply app/templates/components/forms/wizard/attendee-step.hbs:101]: ok

**Comment by ruhci28** [comment]
> Also, please change the PR title to something proper

Yeah sure.

**Comment by ruhci28** [comment]
Thanks @iamareebjamal 🙂

**Pull request merged**